src/routes/follow.ts

```diff
@@ -0,0 +1,45 @@
+import express from "express";
+import {ensureAuthUser} from "@/middlewares/authentication";
+import {createFollow, deleteFollow} from "@/models/follow";
+
+export const followRouter = express.Router();
+
+followRouter.post(
+  "/:userId/follows",
+  ensureAuthUser,
+  async (req, res, next) => {
+    const {userId} = req.params;
+    const redirectUrl = req.query.redirect;
+    const currentUserId = req.authentication?.currentUserId;
+    if (currentUserId === undefined) {
+      // `ensureAuthUser` enforces `currentUserId` is not undefined.
+      // This must not happen.
+      return next(new Error("Invalid error: currentUserId is undefined."));
+    }
+    await createFollow({
+      followingId: Number(currentUserId),
+      followedId: Number(userId),
+    });
+    res.redirect(String(redirectUrl));
+  }
+);
+
+followRouter.delete(
+  "/:userId/follows",
+  ensureAuthUser,
+  async (req, res, next) => {
+    const {userId} = req.params;
+    const redirectUrl = req.query.redirect;
+    const currentUserId = req.authentication?.currentUserId;
+    if (currentUserId === undefined) {
+      // `ensureAuthUser` enforces `currentUserId` is not undefined.
+      // This must not happen.
+      return next(new Error("Invalid error: currentUserId is undefined."));
+    }
+    await deleteFollow({
+      followingId: Number(currentUserId),
+      followedId: Number(userId),
+    });
+    res.redirect(String(redirectUrl));
```

Comment:
最初から文字列なので変換の必要は無さそうです!

```suggestion
    res.redirect(redirectUrl);
```

Reply:
こうしないとエラー出るんですよ

Reply:
お、後で時間あるときに見てみますね!とりあえず今のままでマージしてOKです！